web/src/javascript/game.js

```diff
@@ -105,6 +105,43 @@ export function drawResourcesOnMap(resources, canvas) {
   drawResources(resources.gold, constants.goldColor, constants.goldRGBColor,  constants.resourceMass);
 }
 
+export function drawVirusesOnMap(viruses, canvas) {
+  const context = canvas.getContext("2d");
+
+  for(const virus of viruses) {
+    const grad = context.createRadialGradient(virus.x,virus.y, 5, virus.x, virus.y, constants.virusRadius);
+    grad.addColorStop(0, constants.virusColor);
+    grad.addColorStop(1, constants.virusEndColor);
```

Comment:
Are you sure the constant `virusEndColor` exists? It's popping an error on my side because of this